hash/specpls3_flop.xml

```diff
@@ -18299,14 +18370,15 @@ Disk 3 - Side A: Vindicators (Domark)
 	<software name="tikitaca">
 		<description>Tiki Taca</description>
 		<year>2019</year>
-		<publisher>Climacus / Errazking</publisher>
+		<publisher>Climacus</publisher>
+		<info name="developer" value="Eduardo Martin Torices / Igor Errazkin" />
 		<part name="flop" interface="floppy_3">
 			<dataarea name="flop" size="194816">
 				<rom name="tiki taca (2019)(climacus)(+3)(es)(en).dsk" size="194816" crc="f2e1d3d4" sha1="316de7cd90cb66b1094320d1d65b3fc5eb8543de"/>
 			</dataarea>
 		</part>
 	</software>
-
+	
```

Comment:
Stray whitespace here.  Maybe run srcclean on the file to reduce the chance of churn?